kiosk/src/Models/Kiosk.ts

```diff
@@ -137,6 +137,9 @@ export class Kiosk {
         const addedGames = this.getAllAddedGames();
         const addedGamesObjs: GameData[] = Object.values(addedGames);
         for (const game of addedGamesObjs) {
+            if (!game?.userAdded) {
+                game.userAdded = true;
+            }
             if (!game?.deleted) {
```

Comment:
Going off Joey's suggestion, should it be written this way instead?
```suggestion
            if (game && !game.deleted) {
```

Reply:
This is a good point. I'm over-using optional chaining here and using it for the wrong reasons. Because this loop is iterating over all the games in the list, we shouldn't get a null issue, so I think just having `!game.deleted` would be fine. However, I like the extra protection this adds just in case something goes wrong, so I'll use this.